packages/components/src/TooltipTrigger/TooltipTrigger.component.js

```diff
@@ -80,8 +133,21 @@ class TooltipTrigger extends React.Component {
 		);
 		// TODO jmfrancois : render the Tooltip in a provider so use context for that.
 		return (
-			<OverlayTrigger placement={this.props.tooltipPlacement} overlay={tooltip} delayShow={400}>
-				{this.props.children}
+			<OverlayTrigger
+				ref={ref => {
+					this.overlay = ref;
+				}}
+				placement={this.props.tooltipPlacement}
+				overlay={tooltip}
+				delayShow={400}
+				animation={false}
+			>
+				{cloneElement(child, {
+					onMouseDown: createChainedFunction(this.onMouseDown, childrenProps.onMouseDown),
```

Comment:
Everytime there is a render, a new function props is created. (x the number of callback).
And `createChangedFunction()` is quite complicated for chaining (at most) 2 functions.

Just set the calls in the class methods if the props exist.